src/pages/guides/endpoints/classifications/index.md

```diff
@@ -5,11 +5,11 @@ description: Use Analytics classification APIs to categorize variable data.
 
 # Analytics Classification APIs
 
-The Analytics 2.0 Classification API endpoints allow you to categorize variable data, then display it in different ways when you generate reports. The endpoints use the same data and methods that are used when working with classifications in the UI. See the [Classifications overview](https://experienceleague.adobe.com/docs/analytics/components/classifications/c-classifications.html) for more information.
+The Analytics 2.0 Classification API endpoints allow you to categorize variable data, then display it in different ways when you generate reports. The endpoints use the same data and methods that are used when working with classifications in the Adobe Analytics UI. See the [Classifications overview](https://experienceleague.adobe.com/docs/analytics/components/classifications/c-classifications.html) for more information.
 
-The endpoints described in this guide are routed through analytics.adobe.io. To use them, you will need to first create a client with access to the Adobe Developer Console. For more information, refer to [Getting started with the Analytics API](src/pages/guides/endpoints/classifications/index.md).
+The endpoints described in this guide are routed through `analytics.adobe.io`. You must first create a client with access to the Adobe Developer Console to use them. See [Getting started with the Analytics API](src/pages/guides/endpoints/classifications/index.md) for more information.
 
-Additionally, using these endpoints requires your global company ID in each request. You can find your global company ID by using the [Discovery API](../discovery.md).
+Additionally, using these endpoints requires your global company ID in each request. You can find your global company ID by using the [Discovery API](../discovery.md)
```

Comment:
The periods got mixed up in some of the content in this guide, but in this case the period is needed.